README.md

```diff
@@ -1,11 +1,87 @@
-### DX-MGN-POOL
+# DX-MGN-POOL
 
-The following repo contains all the smart contracts for the pool. Its goal is to collect liquidity that will automatically and continuously trade on the dutch exchange (in form of sell orders). It will thus generate MGN, which the liquidity provider can claim according to their share, once the pooling period has ended.
+The following repo contains all the smart contracts for the pool. Its goal is to collect liquidity that will automatically and continuously trade on the dutch exchange (in form of sell orders) over a predefined amount of time (~1 month). It will thus generate MGN, which the liquidity providers can claim according to their share, once the pooling period has ended.
 
 A rough state diagram of the contract looks like this:
 
 ![dx-mgn-pool state diagram](dx-mgn-pool%20state%20machine.png)
 
+The deployment consists of two smart contracts, `DxMgnPool` and a `Coordinator`
+
+### DxMgnPool
+
+This contract allows participants to deposit a single predefined ERC20 token (e.g. GNO) into the contract's balance. 
+The contract will continuously participate with its entire balance in dutchX auctions and thus generate magnolia.
```

Comment:
(if the token is a whitelisted token); I assume that the contracts also work on none-whitelisted tokens?